backend/app/api/questions_api.py

```
@@ -0,0 +1,111 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.db.session import get_db
+from app.models.questions import QuestionCreate, QuestionUpdate, QuestionRead
+from app.services import questions_services
+from app.utils.responses import get_responses
```

Flake8 (7.2.0)

[error] 'app.utils.responses.get_responses' imported but unused

(F401)